src/components/AddBook.js

```diff
@@ -1,16 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
 
-const AddBook = () => (
-  <div className="addbook-content">
-    <h4>ADD NEW BOOK</h4>
-    <div className="add-book-btns">
-      <form>
-        <input type="text" />
-        <input type="text" />
-        <input type="submit" value="ADD BOOK" />
+const AddBook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
 
-      </form>
+  const submitBookToStore = (e) => {
+    e.preventDefault();
+
+    const newBook = {
+      id: 4, // make sure it's unique
+      title,
+      categorie: 'Action',
+      author: 'example',
```

Comment:
- [ ] It will be great to add the author from your state. This will make sure that the user can be able to add the name of each author of a certain book. This is a major requirement for your project to be approved